.travis.yml

```diff
@@ -111,8 +111,8 @@ install:
         && brew link --overwrite gcc;
     fi
 # OCCA
+  # TODO(dmed256): Use a specific commit/tag (cd occa && git reset --hard [COMMITISH] && cd ..)
```

Comment:
We weren't using any specific commit for a while, so it should be ok to leave this without a commit, in my opinion